web-app/js/portal/details/InfoPanel.js

```diff
@@ -26,20 +26,54 @@ Portal.details.InfoPanel = Ext.extend(Ext.Container, {
     },
 
     _initWithLayer: function() {
-        var metadataUrl = 'layer/getFormattedMetadata?uuid=' +
+
+        var metadataUrl = 'layer/getMetadataAbstract?uuid=' +
             encodeURIComponent(this.layer.metadataUuid);
 
         Ext.Ajax.request({
             url: metadataUrl,
             scope: this,
             success: function(resp, options) {
-                this.update(resp.responseText, false);
+
+                this._constructInfoTabHtml(resp.responseText, this.layer.parentGeoNetworkRecord.data.onlineResources);
             },
             failure: function(resp) {
                 this.update("<i>" + OpenLayers.i18n('noMetadataMessage') + "</i>", false);
```

Comment:
So this is where you should return the error to the user, in `noMetadataMessage`.